src/test/e2e-emulated/map.spec.ts

```diff
@@ -1307,4 +1307,6345 @@ describe("map", () => {
             throw e;
         }
     });
+
+    it("should implement map.replace correctly", async () => {
+        jest.setTimeout(2 * 60000);
+        try {
+            // Initial state
+            expect((await contract.getIntMap1()).size).toBe(0);
+            expect((await contract.getIntMap2()).size).toBe(0);
+            expect((await contract.getIntMap3()).size).toBe(0);
+            expect((await contract.getIntMap4()).size).toBe(0);
+            expect((await contract.getIntMap5()).size).toBe(0);
+            expect((await contract.getIntMap6_1()).size).toBe(0);
+            expect((await contract.getIntMap6_2()).size).toBe(0);
+            expect((await contract.getIntMap6_3()).size).toBe(0);
+            expect((await contract.getIntMap6_4()).size).toBe(0);
+            expect((await contract.getIntMap6_5()).size).toBe(0);
+            expect((await contract.getIntMap6_6()).size).toBe(0);
+            expect((await contract.getIntMap6_7()).size).toBe(0);
+            expect((await contract.getIntMap7_1()).size).toBe(0);
+            expect((await contract.getIntMap7_2()).size).toBe(0);
+            expect((await contract.getIntMap7_3()).size).toBe(0);
+            expect((await contract.getIntMap7_4()).size).toBe(0);
+            expect((await contract.getIntMap7_5()).size).toBe(0);
+            expect((await contract.getIntMap7_6()).size).toBe(0);
+            expect((await contract.getIntMap8_1()).size).toBe(0);
+            expect((await contract.getIntMap8_2()).size).toBe(0);
+            expect((await contract.getIntMap8_3()).size).toBe(0);
+            expect((await contract.getIntMap8_4()).size).toBe(0);
+            expect((await contract.getIntMap8_5()).size).toBe(0);
+            expect((await contract.getIntMap8_6()).size).toBe(0);
+            expect((await contract.getIntMap8_7()).size).toBe(0);
+            expect((await contract.getIntMap9_1()).size).toBe(0);
+            expect((await contract.getIntMap9_2()).size).toBe(0);
+            expect((await contract.getIntMap9_3()).size).toBe(0);
+            expect((await contract.getIntMap9_4()).size).toBe(0);
+            expect((await contract.getIntMap9_5()).size).toBe(0);
+            expect((await contract.getIntMap9_6()).size).toBe(0);
+            expect((await contract.getIntMap10_1()).size).toBe(0);
+            expect((await contract.getIntMap10_2()).size).toBe(0);
+            expect((await contract.getIntMap10_3()).size).toBe(0);
+            expect((await contract.getIntMap10_4()).size).toBe(0);
+            expect((await contract.getAddrMap1()).size).toBe(0);
+            expect((await contract.getAddrMap2()).size).toBe(0);
+            expect((await contract.getAddrMap3()).size).toBe(0);
+            expect((await contract.getAddrMap4()).size).toBe(0);
+            expect((await contract.getAddrMap6_1()).size).toBe(0);
```

Comment:
Is there a reason for the missing getAddrMap5?

Reply:
same as below